Automattic/wp-calypso
# [Domains] Remove obsolete components

Follow-up of #24609, which enabled the redesign in the domain management page (`/domain/add`). This change removes the errant domain flags that appear on the domain management page. 

The precursor to this change is #24341.

# Test Instructions
1. Spin up this branch locally or fire up a [live branch](https://calypso.live/?branch=remove/unused-domains-components).
2. Navigate to `/domains/add` and enter a query (i.e. `spirit yoga design`)
3. Ensure that no domain flags render on the page.

<img width="1059" alt="domain_search_ _testing_ _wordpress_com" src="https://user-images.githubusercontent.com/4044428/39549909-b7813b00-4e1c-11e8-800d-697ae2befa53.png">
